libs/cart/testing/src/drivers/in-memory/cart-shipping-address/cart-shipping-address.service.ts

```diff
@@ -0,0 +1,26 @@
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { Observable } from 'rxjs';
+
+import {
+  DaffCart,
+  DaffCartShippingAddressServiceInterface,
+  DaffCartAddress
+} from '@daffodil/cart';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class DaffInMemoryCartShippingAddressService implements DaffCartShippingAddressServiceInterface<DaffCartAddress, DaffCart> {
+  url = '/api/cart-shipping-address';
+
+  constructor(private http: HttpClient) {}
+
+  get(cartId: DaffCart['id']): Observable<DaffCartAddress> {
+    return this.http.get<DaffCartAddress>(`${this.url}/${cartId}`);
+  }
+
+  update(cartId: DaffCart['id'], address: DaffCartAddress): Observable<Partial<DaffCart>> {
```

Comment:
this should take `address: Partial<DaffCartAddress>`

Reply:
Changed